README.md

```diff
@@ -375,6 +375,12 @@ In order of preference, the plugin determines what command to invoke for `rustc`
 
 (Note that failure to locate `rustc` is not fatal, however it may result in rebuilding the code more often than is necessary).
 
+In order of preference, the plugin determines what toolchain to invoke for `rustup` by:
+
+1. `rust.rustupToolchain` in `${rootDir}/local.properties`
+1. the environment variable `RUST_TOOLCHAIN_VERSION`
+1. the default, `stable`
```

Comment:
This is a little misleading, 'cuz we don't actually _specify_ `stable`.

Also, if I remember the syntax, it's `cargo +nightly ...` or whatever.  Can we save consumers a round of trial and error by including the `+...` in an example here in the documentation, or by ensuring there's a leading `+` in the code itself?